tools/pngquant/README.txt

```diff
@@ -1,6 +1,6 @@
 # pngquant 2
 
-[pngquant](https://pngquant.org) is a PNG compresor that significantly reduces file sizes by converting images to a more efficient 8-bit PNG format *with alpha channel* (often 60-80% smaller than 24/32-bit PNG files). Compressed images are fully standards-compliant and are supported by all web browsers and operating systems.
```

Comment:
This is a dependency that is copied from the source. Please do not change these files, as it will make upgrading dependencies harder.